js/runtime.js

```diff
@@ -21,9 +21,11 @@
     }
 
     function getContext(){
+        // returns CanvasContext
         if (!_ctx){
             _ctx = canvas().getContext('2d');
             // Save the default state.
+            _ctx.strokeStyle = 'transparent';
```

Comment:
strokeStyle defaults to "#000" in the Canvas API, but in Waterbear we don't want a strokeStyle unless the user sets one.